planemo/options.py

```diff
@@ -177,7 +177,7 @@ def galaxy_python_version():
         "--galaxy_python_version",
         use_global_config=True,
         default=None,
-        type=click.Choice(["3", "3.7", "3.8", "3.9", "3.10", "3.11"]),
+        type=click.Choice(["3.8", "3.9", "3.10", "3.11", "3.12"]),
```

Comment:
3.13 can be used with the upcoming Galaxy release 24.2 .

Reply:
```suggestion
        type=click.Choice(["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]),
```